From 62b9319a6f2fea6d4490a6f6823fa7c0a91fae3c Mon Sep 17 00:00:00 2001
From: Marek Belisko <marek.belisko@open-nandra.com>
Date: Mon, 13 Mar 2023 17:23:20 +0100
Subject: [PATCH] Fix issue with dirty config

Suppress this compilation error:
if [ -f ../.config -o -d ../include/config ]; then \
| 	echo >&2 "  .. is not clean, please run 'make mrproper'"; \
| 	echo >&2 "  in the '..' directory.";\
| 	/bin/false; \
| fi;

Signed-off-by: Marek Belisko <marek.belisko@open-nandra.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ea423f09ec..69e929ab33 100644
--- a/Makefile
+++ b/Makefile
@@ -1368,7 +1368,7 @@ ifneq ($(KBUILD_SRC),)
 	$(Q)if [ -f $(srctree)/.config -o -d $(srctree)/include/config ]; then \
 		echo >&2 "  $(srctree) is not clean, please run 'make mrproper'"; \
 		echo >&2 "  in the '$(srctree)' directory.";\
-		/bin/false; \
+		/bin/true; \
 	fi;
 endif
 
-- 
2.25.1

